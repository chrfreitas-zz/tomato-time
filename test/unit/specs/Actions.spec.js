import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils'

import Actions from '../../../src/modules/timer/components/actions/actions.component';

const localVue = createLocalVue()
localVue.use(Vuex)

describe('Timer:Actions', () => {
  let getters
  let store

  beforeEach(() => {
    getters = {
      isDisabledNext: jest.fn(),
      isDisabledPrevious: jest.fn()
    }
    store = new Vuex.Store({
      getters
    })
  })

  it('should create/match snapshot', () => {
    const wrapper = shallowMount(Actions, {store, localVue});
    expect(wrapper.html()).toMatchSnapshot();
  });
});
