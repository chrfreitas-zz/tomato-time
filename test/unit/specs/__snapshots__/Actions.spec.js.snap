// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer:Actions should create/match snapshot 1`] = `
<div class="actions">
    <button class="btn">
        <i class="fas fa-angle-left"></i>
    </button>
    <counter-stub></counter-stub>
    <button class="btn">
        <i class="fas fa-angle-right"></i>
    </button>
</div>
`;
